stacker/lookups/handlers/dynamodb.py

```diff
@@ -4,63 +4,45 @@
 
 from ...util import read_value_from_path
 
-TYPE_NAME = "dynamodb"
+TYPE_NAME = 'dynamodb'
 
 
 def handler(value, **kwargs):
     """Get a value from a dynamodb table
 
     dynamodb field types should be in the following format:
 
-        [<region>:<tablename>@]<primarypartionkey>.<keyvalue>.<keyvalue>...
+        [<region>:]<tablename>@<primarypartionkey>:<keyvalue>.<keyvalue>...
 
     Note: The region is optional, and defaults to the environment's
     `AWS_DEFAULT_REGION` if not specified.
-
-
     """
     value = read_value_from_path(value)
     table_info = None
     table_keys = None
     region = None
     table_name = None
-
-    if "@" in value:
-        table_info, table_keys = value.split("@", 1)
-        if ":" in table_info:
-            region, table_name = table_info.split(":", 1)
+    if '@' in value:
+        table_info, table_keys = value.split('@', 1)
+        if ':' in table_info:
+            region, table_name = table_info.split(':', 1)
         else:
             table_name = table_info
     else:
         raise ValueError('Please make sure to include a tablename')
 
-    if table_name is None:
+    if table_name is '':
```

Comment:
Maybe it's better to just use:

```
if not table_name:
```

That way it'll fail for any non-existant value, None or ''.